mmaction/datasets/base.py

```diff
@@ -83,6 +83,15 @@ def __init__(self,
         self.video_infos = self.load_annotations()
         if self.sample_by_class:
             self.video_infos_by_class = self.parse_by_class()
+            class_prob = []
+            for k, samples in self.video_infos_by_class.items():
```

Comment:
```python
class_nums = np.array(list(self.video_infos_by_class.values()))
class_prob = class_nums / max(1, class_nums.sum())
```

Reply:
or

```python
num = max(1, sum(self.video_infos_by_class.values()))
self.class_prob = {k:v/num for k, v in self.video_infos_by_class.items()}
```

Reply:
I do this to avoid lines exceeding 80 characters, which will look bad.

Reply:
pre-commit will do the formatting